packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/PageContent.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {
+  Field,
+  RichText as JssRichText,
+  useSitecoreContext,
+  LayoutServiceContext,
+  RouteData
+} from '@sitecore-jss/sitecore-jss-nextjs';
+
+export type SitecoreContextValue = LayoutServiceContext & {
```

Comment:
We have the same type [here](https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-react/src/components/SitecoreContext.tsx#L25) can be used